### DISCUSSION TITLE
How to match next bracket in shortcut?

### DISCUSSION THREAD

**Discussion by Dennis-Zhang-SH** [ayamir/nvimdots#793] [Q&A] [answered]
How to match next bracket in shortcut?

**Comment by ayamir** [comment]
What's the meaning of "match"?

**Reply by Dennis-Zhang-SH** [reply]
> What's the meaning of "match"?

I mean navigate cursor to next matched bracket.

**Reply by charliie-dev** [reply]
have u searched for vim motions?

**Reply by Dennis-Zhang-SH** [reply]
> have u searched for vim motions?

I searched, the community says just search the next bracket, I wonder is there shortcut to do such thing.

**Reply by ayamir** [reply]
`%` is the default way that vim does this in normal mode.
`<A-h>` and `<A-l>` can be used in insert mode to move cursor to the left and the right bracket.

**Comment by kang8** [comment, marked as answer]
If you use [andymass/vim-matchup](https://github.com/andymass/vim-matchup), you can press `%`(or press twice) to match next matched word(it can be bracket).

![2023-06-08 18 12 22](https://github.com/ayamir/nvimdots/assets/36906329/faaa685a-1355-41a2-b04c-0dc9266ffdc7)
